Orthtree/include/CGAL/Orthtree.h

```diff
@@ -468,14 +465,16 @@ class Orthtree {
     \return the bounding box of the node n
    */
   Bbox bbox(Node_index n) const {
-
     using Cartesian_coordinate = std::array<FT, Dimension::value>;
     Cartesian_coordinate min_corner, max_corner;
-    Bbox_dimensions size = m_side_per_depth[depth(n)];
-
+    std::size_t node_depth = depth(n);
+    Bbox_dimensions size = m_side_per_depth[node_depth];
+    const std::size_t last_coord = std::pow(2,node_depth)-1;
     for (int i = 0; i < Dimension::value; i++) {
-      min_corner[i] = approx(m_bbox_min[i] + int(global_coordinates(n)[i]) * size[i]).inf();
-      max_corner[i] = approx(m_bbox_min[i] + int(global_coordinates(n)[i] + 1) * size[i]).sup();
+      min_corner[i] = (m_bbox.min)()[i] + int(global_coordinates(n)[i]) * size[i];
+      max_corner[i] = std::size_t(global_coordinates(n)[i]) == last_coord
+                    ? (m_bbox.max)()[i]
+                    : (m_bbox.min)()[i] + int(global_coordinates(n)[i] + 1) * size[i];
```

Comment:
I don't think that this will guarantee that neighboring nodes of different depth in the orthtree will have an exact shared coplanar side.